trompace/queries/mediaobject.py

```diff
@@ -0,0 +1,64 @@
+# Generate GraphQL queries for queries pertaining to media objects.
+from trompace.exceptions import UnsupportedLanguageException, NotAMimeTypeException
+from trompace.queries.templates import format_query
+from trompace import StringConstant, _Neo4jDate, filter_none_args
+from trompace.constants import SUPPORTED_LANGUAGES
+
+
+def query_mediaobject(identifier: str=None, name: str=None, description: str=None, date: str=None, creator: str=None, contributor: str=None, format_: str=None,\
+ encodingFormat: str=None, source: str=None, subject: str=None, contentUrl:str=None, language: str=None, title:str=None):
```

Comment:
I don't think that all of these methods need to be available for a query. We don't yet have a clear idea of the places that this method might be used, so we don't really know what we'll need, but from looking at the fields I think that these ones are important:
 * identifier
 * creator
 * contributor
 * encodingFormat
 * source
 * contenturl

These are what I would consider the "stable" identifiers - ids, sources, and people. 
We should separately add a "search" method which allows us to query by title and name.